apps/web/src/routes/Messages/MessageSection/MessageSection.js

```diff
@@ -120,7 +157,10 @@ export default class MessageSection extends React.Component {
   detectScrollExtremes = throttle(target => {
     if (this.props.pending) return
     // Marks entire thread as read if we've seen the last message
-    if (this.atBottom(target)) this.markAsRead()
+    if (this.atBottom(target)) {
+      this.markAsRead()
```

Comment:
need to also call markAsRead when loading the message if the bottom most message is visible (only a couple messages for example)

Reply:
Ok